src/index.js

```diff
@@ -763,11 +763,11 @@ export class ULabel {
         `)
 
         return {
-            "container_id": arguments[0],  // Required
-            "image_data": arguments[1],    // Required
-            "username": arguments[2],      // Required
-            "submit_butons": arguments[3], // Required
-            "subtasks": arguments[4],      // Required
+            "container_id": arguments[0],   // Required
+            "image_data": arguments[1],     // Required
+            "username": arguments[2],       // Required
+            "submit_buttons": arguments[3], // Required
```

Comment:
since this wasn't a argument previously, does it even make sense to expect it along with the other deprecated args?

Reply:
The `on_submit` argument is the `submit_buttons` argument. In the past only one submit button was supported, but when we expanded support for multiple submit buttons (e.g. a submit and a reject button) everything internally was renamed from `on_submit` to `submit_buttons`. Looks like the api_spec was never updated. Nice catch 👍

Reply:
oh i see, that makes a lot of sense actually 🤯